src/book/entities/Account.ts

```diff
@@ -71,24 +69,7 @@ export default class Account extends BaseEntity {
   @v.IsIn(Account.TYPES)
     type!: string;
 
-  @BeforeInsert()
-  @BeforeUpdate()
-  async setPath() {
-    if (this.type === 'ROOT' || this.parent?.type === 'ROOT') {
-      this.path = this.name;
-    } else {
-      // havent found a way to control BeforeInsert order.
-      // The validation one executes after this so we have to protect
-      // against parent not existing (which is mandatory).
-      this.path = `${this.parent?.path}:${this.name}`;
-    }
-  }
-
-  @Column({
-    type: 'text',
-    default: '',
-  })
-    path!: string;
```

Comment:
path is not a DB column anymore but a dynamically computed field when calling `useAccounts`.